v0/tests/BecomingTypeAdvancedTest.php

```
@@ -0,0 +1,386 @@
+                    }
+
+                    public function offsetGet($offset): mixed
+                    {
+                        return null;
+                    }
+
+                    public function offsetSet($offset, $value): void
+                    {
+                    }
+
+                    public function offsetUnset($offset): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$offset'. (undefined)

(UnusedFormalParameter)